airflow/providers/databricks/hooks/databricks.py

```diff
@@ -63,6 +64,23 @@
 SPARK_VERSIONS_ENDPOINT = ("GET", "api/2.0/clusters/spark-versions")
 
 
+class RunLifeCycleState(Enum):
```

Comment:
This seems to be a duplication of https://github.com/apache/airflow/blob/4e0f9ddd710c47e8ef93b523c5b8df957d14a315/airflow/providers/databricks/hooks/databricks.py#L87-L95. Should we remove `RUN_LIFE_CYCLE_STATES`? or point it to `RunLifeCycleState`

Reply:
or maybe this is a refactor we can have in the next PR

Reply:
yes, that could affect pieces of code in other operators. I think good candidate for another PR focussed on using the Enum in all operators.